housekeeper/cli/delete.py

```diff
@@ -134,6 +134,28 @@ def files_cmd(context, yes, tag, bundle_name, before, notondisk, list_files, lis
             delete_file(file=file, store=store)
 
 
+@delete.command("tag")
+@click.option("-n", "--name", help="Name of the tag to delete")
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.pass_context
+def tag_cmd(context, yes, name: str):
+    """Delete a tag from the database.
+    Entries in the file_tag_link table associated with the tag will be removed."""
+    store: Store = context.obj["store"]
+    tag: Tag = store.get_tag(tag_name=name)
+    if not tag:
+        LOG.warning(f"Tag {name} not found")
+        raise click.Abort
+
+    tag_files: int = len(tag.files) if tag.files else 0
```

Comment:
Naming suggestion, feel free to disregards. I think it is nice to have `counter` or `count` in variables keeping count of stuff

Reply:
changed